orcestra/flightplan.py

```diff
@@ -201,14 +214,15 @@ def __call__(self, start: LatLon, include_start: bool = False):
                     f"could not find a solution for circle {self} with start point {start}, maybe start is inside cirle. You may want to try without specifying `enter`."
                 )
 
-        angles = np.linspace(start_angle, start_angle + self.angle, 30)
+        n_points = 30
```

Comment:
This could be an attribute of the class, or an module-wide parameter that users can set?

Reply:
Yes, there should be a way of customizing this. But I don't yet know how to do it best. I'd for now either
* leave it as is  (to still document the purpose of this parameter) or
* undo the change, to keep it out of this PR (as it's somewhat unrelated)

Reply:
I am fine with leaving it as is for documentation purposes

Reply:
I've just opted to leave it out. It distracts from this PR and can't be solved in a good way quickly.